std_msgs/Header header
geometry_msgs/Quaternion orientation
geometry_msgs/Vector3 omega
geometry_msgs/Vector3 gyro_bias
geometry_msgs/Vector3 accel_bias
